examples/complete_parser.cpp

```diff
@@ -54,97 +54,97 @@ int main(int argc, char* argv[])
         petro::box::ftyp,
         petro::box::pdin,
         petro::box::moov<petro::parser<
-            petro::box::mvhd,
-            petro::box::trak<petro::parser<
-                petro::box::tkhd,
-                petro::box::tref,
-                petro::box::edts<petro::parser<
-                    petro::box::elst
-                >>,
-                petro::box::mdia<petro::parser<
-                    petro::box::mdhd,
-                    petro::box::hdlr,
-                    petro::box::minf<petro::parser<
-                        petro::box::vmhd,
-                        petro::box::smhd,
-                        petro::box::hmhd,
-                        petro::box::nmhd,
-                        petro::box::dinf<petro::parser<
-                            petro::box::dref<petro::parser<
-                                petro::box::url,
-                                petro::box::urn
-                            >>
+                petro::box::mvhd,
```

Comment:
That is the expected behaviour, and I also mentioned that example here: http://docs.steinwurf.com/coding_style.html#template-expressions
One level of indentation (4 spaces) for a single < opener, so two indents are added if a line has two < openers.